addons/website_forum/models/forum.py

```diff
@@ -129,6 +131,23 @@ def _get_default_faq(self):
     karma_post = fields.Integer(string='Ask questions without validation', default=100)
     karma_moderate = fields.Integer(string='Moderate posts', default=1000)
 
+    @api.depends('post_ids')
+    def _compute_last_post_id(self):
+        for forum in self:
+            forum.last_post_id = forum.post_ids.search([('parent_id', '=', False)], order='create_date desc', limit=1)
+
+    @api.depends('description')
+    def _compute_teaser(self):
+        for forum_post in self:
+            if forum_post.description:
```

Comment:
forum_post ???

don't forget migration script !!

Reply:
Ok, I changed it in "forum", but what would happen with migration if not ?

Reply:
migration is another comment...

When you add new field in a PR, you need to have relate PR for migration repo which add this field.

Reply:
Didn't know about that, so, how it works ? I need a PR from a new branch, created only for the field ? And what about the view that use this field ?